package.json

```diff
@@ -10,6 +10,9 @@
   "bin": {
     "bondage": "src/bin/bondage.js"
   },
+  "scripts": {
```

Comment:
The build works as a charm now. Btw you already have an entry for scripts in package.json on line 6

Reply:
lol thanks 🤦‍♀